src/pom.xml

```diff
@@ -20,6 +20,7 @@
     <properties>
         <java.version>1.8</java.version>
         <springfox-swagger-version>2.6.1</springfox-swagger-version>
+        <sonar.jacoco.reportPath>${project.basedir}/../target/jacoco.exec</sonar.jacoco.reportPath>
```

Comment:
What is jacoco?

Reply:
Jacoco is a plugin that calculate the code coverage based on our unit test, the report can be then uploaded to sonarqube to track our coverage

Reply:
OK, thanks for the explanation.